index.html

```diff
@@ -48,8 +48,17 @@ <h1 class="flex flex-wrap">
         </div>
     </header>
 
-    <div class="wrapper">
+    <noscript>
+        <div id="noscript" class="d-flex align-items-center flex-column justify-content-center">
+            <p>
+                Pour pouvoir utiliser ce générateur, il vous faut préalablement activer JavaScript.<br />
+                <a href="https://www.enable-javascript.com/fr/">Cliquez ici</a> pour accéder aux instructions
+                pour activer JavaScript dans votre navigateur.
```

Comment:
```suggestion
                <a href="https://www.enable-javascript.com/fr/">Cliquez ici pour accéder aux instructions
                pour activer JavaScript dans votre navigateur</a>.
```
Pour améliorer l'accessibilité.

Reply:
Je m'étais posé la question en effet, pour des raisons esthétiques je ne l'avais pas fait mais oui tu as raison.